src/services/validator_state.py

```diff
@@ -42,7 +42,7 @@ def get_extra_data(self, blockstamp: ReferenceBlockStamp, chain_config: ChainCon
             stuck_validators=stuck_validators,
             exited_validators=exited_validators,
             max_items_in_payload_count=orl.max_accounting_extra_data_list_items_count,
-            max_items_count=orl.max_accounting_extra_data_list_items_count,
+            max_items_count=orl.max_node_operators_per_extra_data_item_count,
```

Comment:
pls double check it, it looks like we need to swap them

Reply:
Fixed